index.bs

```diff
@@ -1236,15 +1282,87 @@ To <dfn>check if an [=attribution source=] can create [=aggregatable contributio
     |remainingAggregatableBudget|, return false.
 1. Return true.
 
-<h3 algorithm id="triggering-event-level-attribution">Triggering event-level attribution</h3>
+<h3 id="obtaining-trigger-debug-data">Obtaining debug data on trigger registration</h3>
+
+To <dfn>obtain debug data body on trigger registration</dfn> given a
+[=trigger debug data type=] |dataType|, an [=attribution trigger=] |trigger|,
+an optional [=attribution source=] <dfn for="obtain debug data body on trigger registration">
+<var>sourceToAttribute</var></dfn>, and an optional [=attribution report=]
+<dfn for="obtain debug data body on trigger registration"><var>report</var></dfn>:
+
+1. Let |body| be a new [=map=] with the following key/value pairs:
+    : "`attribution_destination`"
+    :: |trigger|'s [=attribution trigger/attribution destination=], <a href="https://html.spec.whatwg.org/multipage/origin.html#serialization-of-a-site">serialized</a>.
+
+1. If |trigger|'s [=attribution trigger/debug key=] is not null, [=map/set=] |body|["`trigger_debug_key`"]
+    to |trigger|'s [=attribution trigger/debug key=], [=serialize an integer|serialized=].
+1. If |sourceToAttribute| is not null:
+    1. [=map/Set=] |body|["`source_event_id`"] to |source|'s [=attribution source/event ID=], [=serialize an integer|serialized=].
+    1. [=map/Set] |body|["`source_site`"] to |source|'s [=attribution source/source site=], <a href="https://html.spec.whatwg.org/multipage/origin.html#serialization-of-a-site">serialized</a>.
+    1. If |sourceToAttribute|'s [=attribution source/debug key=] is not null, [=map/set=]
+        |body|["`source_debug_key`"] to |sourceToAttribute|'s [=attribution source/debug key=],
+        [=serialize an integer|serialized=].
+
+1. If |dataType| is:
+    <dl class="switch">
+    <dt>"`trigger-reporting-origin-limit`"</dt>
+    <dd>[=map/Set=] |body|["`limit`"] to [=max attribution reporting endpoints per rate-limit window=],
+         [=serialize an integer|serialized=].</dd>
+    <dt>"trigger-attributions-per-source-destination-limit"</dt>
+    <dd>[=map/Set=] |body|["`limit`"] to [=max attributions per rate-limit window=],
+         [=serialize an integer|serialized=].</dd>
+    <dt>"`trigger-event-low-priority`"</dt>
+    <dt>"`trigger-event-excessive-reports`"</dt>
+    <dd>
+        1. [=Assert=]: |report| is not null and is an [=event-level report=].
+        1. Return the result of running [=obtain an event-level report body=] with |report|.
```

Comment:
To be clear: We are returning the actual event-level report in this case?

Reply:
Yes, the implementation is here (https://source.chromium.org/chromium/chromium/src/+/main:content/browser/attribution_reporting/attribution_debug_report.cc;l=296).